test/extended/util/cluster/cluster.go

```diff
@@ -92,6 +99,12 @@ func DiscoverClusterState(clientConfig *rest.Config) (*ClusterState, error) {
 
 	state := &ClusterState{}
 
+	url, _, err := rest.DefaultServerURL(clientConfig.Host, clientConfig.APIPath, schema.GroupVersion{}, false)
+	if err != nil {
+		return nil, err
+	}
+	state.APIURL = url
```

Comment:
@deads2k Needs a lgtm again. I had this logic reversed and it was causing a panic :facepalm: